src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -222,19 +224,19 @@ private JsonConverter GetConverterFromAttribute(JsonConverterAttribute converter
             return converter;
         }
 
-        private static Attribute GetAttributeThatCanHaveMultiple(Type classType, Type attributeType, PropertyInfo propertyInfo)
+        private static Attribute? GetAttributeThatCanHaveMultiple(Type classType, Type attributeType, PropertyInfo propertyInfo)
         {
-            object[] attributes = propertyInfo?.GetCustomAttributes(attributeType, inherit: false);
+            object[] attributes = propertyInfo.GetCustomAttributes(attributeType, inherit: false);
```

Comment:
This method only called when propertyInfo is not null so removed `?`

Reply:
Makes sense to me.